components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -239,17 +298,22 @@ public void testChownAnnotationPrivate(boolean isDataOwner, boolean isAdmin, boo
         /* note which objects were used to annotate an image */
 
         final List<IObject> imageAnnotations;
+        final List<IObject> imageAnnotations2;
         final List<ImageAnnotationLink> tagLinksOnOtherImage = new ArrayList<ImageAnnotationLink>();
-
-        /* import and annotate an image */
+        final List<ImageAnnotationLink> fileAnnLinksOnOtherImage = new ArrayList<ImageAnnotationLink>();
+        final List<ImageAnnotationLink> mapAnnLinksOnOtherImage = new ArrayList<ImageAnnotationLink>();
+        
+        /* import and annotate an image with two sets of annotations */
 
         init(importer);
         final Image image = (Image) iUpdate.saveAndReturnObject(mmFactory.createImage()).proxy();
         final long imageId = image.getId().getValue();
         testImages.add(imageId);
         imageAnnotations = annotateImage(image);
+        imageAnnotations2 = annotateImage(image);
```

Comment:
could there be a more descriptive way of naming here, like if some are for double or single linking or whatever?